### PR TITLE
Add page for isolated market trading.

### PR DESCRIPTION
**Pull request by vincentwschau**

### CODE REVIEW

File: pages/api_integration-trading/isolated_markets.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+>Note: This document covers how the feature works from the protocol point of view and not the front-end or the indexer.
+
+# Querying for Isolated Markets
+
+There is a new `market_type` parameter in the `PerpetualParams` proto struct that indicates the type of market. 
+
+There are 2 possible values for this parameter:
+
+- `PERPETUAL_MARKET_TYPE_CROSS` - markets where subaccounts can have positions cross-margined with other `PERPETUAL_MARKET_TYPE_CROSS` markets, all markets created before the v5.0.0 upgrade are `PERPETUAL_MARKET_TYPE_CROSS` markets
+- `PERPETUAL_MARKET_TYPE_SOLATED` - markets where subaccounts can only have a single position in, no cross-margining is possible
```
Comment (shrenujb) [inline pages/api_integration-trading/isolated_markets.md:14]: *ISOLATED

File: pages/api_integration-trading/isolated_markets.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    },
+    "funding_index": "0",
+    "open_interest": "0"
+  }
+}
+```
+
+# Trading Isolated Markets
+
+Positions in isolated markets can only be opened on a subaccount with no open perpetual positions. Once a perpetual position for an isolated market is opened on a subaccount, no positions in any other markets can be opened until the perpetual position is closed.
```
Comment (shrenujb) [inline pages/api_integration-trading/isolated_markets.md:60]: Positions on an isolated market can also be opened on subaccounts with already open positions in the same isolated market
Maybe add that in or word the first line differently

File: pages/api_integration-trading/isolated_markets.md (hunk 1 above)
Comment (shrenujb) [inline pages/api_integration-trading/isolated_markets.md:14]: Maybe describe this as
"Markets that can only be margined in isolated. No cross-margining with other markets is possible"

File: pages/api_integration-trading/isolated_markets.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      "liquidity_tier": 2,
+      "market_type": "PERPETUAL_MARKET_TYPE_ISOLATED"
+    },
+    "funding_index": "0",
+    "open_interest": "0"
+  }
+}
+```
+
+# Trading Isolated Markets
```
Comment (ttl33) [inline pages/api_integration-trading/isolated_markets.md:58]: I think this is more important than the "query" section. Should we move this up?

File: pages/api_integration-trading/isolated_markets.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+# Isolated Markets
```
Comment (ttl33) [inline pages/api_integration-trading/isolated_markets.md:1]: Should we have an overview of what "isolated market" is used for? I think this blog post has a good summary of TLDR, what isolated market is used for, how the insurance fund and collateral pools are separate, etc.

https://dydx.exchange/blog/introducing-isolated-markets-and-isolated-margin

**Review by shrenujb — APPROVED** [review]

**Pull request merged**